base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/dialogs/EditConfigDialog.java

```diff
@@ -57,6 +58,13 @@ public class EditConfigDialog extends ConfigDetailsDialog {
 	private boolean calledSwitchConfigOnSaveAs = false;
 	private String pvName;
 	private ConfigServer server = Configurations.getInstance().server();
+	
+	private static final String CONFIG_HELP_LINK = "https://github.com/ISISComputingGroup/ibex_user_manual/wiki/Create-And-Manage-Configurations";
+	private static final String COMPONENT_HELP_LINK = "https://github.com/ISISComputingGroup/ibex_user_manual/wiki/Create-and-Manage-Components";
+	private final String description;
+	
+	@SuppressWarnings("unused")
+	private HelpButton helpButton;
```

Comment:
If this help button is marked as unused, do we really need it as a member variable? Same for all help buttons.